### PR TITLE
Changes for resetting password

### PR DESCRIPTION
**Pull request by sawyer1997**
- Added backend changes for enabling password reset.
- Made changes so that intermediate sites remain consistent with the theme of our home page.
- The current flow is:
  - User clicks on Forgot Password link
  - User is redirected to a page where he/she needs to fill their email id.
  - If there is any user registered with provided email, he/she will be receive an email (from `soloconnect4@gmail.com`)with a one time password reset link. If they're not registered, Django won't let them know, because it's a potential security leak for hackers.
  - Users after clicking the link provided to them on their email, will be redirected to a page, where password will be asked to fill twice. Upon submitting it, their password will be reset to the new password.
- Updated env variables to `integration` and `prod` AWS envs.

### CODE REVIEW

File: soloconnect/settings.py [hunk 1]
```diff
@@ -138,3 +138,11 @@
 LOGIN_REDIRECT_URL = "user_profile:view_profile"
 
 CRISPY_TEMPLATE_PACK = "bootstrap4"
+# Email configs
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = "smtp.gmail.com"
+EMAIL_USE_TLS = True
+EMAIL_USE_SSL = False
+EMAIL_HOST_USER = "soloconnect4@gmail.com"
+EMAIL_HOST_PASSWORD = "boyp dmed papx airi"
```
Comment (kravisankaran) [inline soloconnect/settings.py:147]: Do we want to hardcode this here - we should ideally add it to AWS configuration and source it as an environment variable

**Review by kravisankaran — CHANGES_REQUESTED** [review]
Have added a comment

File: soloconnect/settings.py (hunk 1 above)
Comment (sawyer1997) [reply soloconnect/settings.py:147]: Yeah, we can do that. But this is not the real password, also, 2FA is turned on for this email account.

**Review by mike-zaharchenko17 — APPROVED** [review]
Lgtm

**Review by kravisankaran — APPROVED** [review]
looks good to merge post env changes

**Pull request merged**